trp_use_case/model/use_case.py

```diff
@@ -175,23 +177,31 @@ def _get_hours_total(self):
             values = {
                 'hours_total': 0.0,
                 'hours_total_optional': 0.0,
-                }
+                'tot_use_cases': 0
+            }
             for use_case in collection.use_case_ids:
                 if use_case.active:
                     values['hours_total'] += use_case.hours
                     values['hours_total_optional'] += use_case.hours_optional
-            values['hours_total_nonoptional'] = (
-                values['hours_total'] - values['hours_total_optional'])
-            # TODO: why doesn't this work?
-            # collection.write(values)
-            collection.hours_total = values['hours_total']
-            collection.hours_total_optional = values['hours_total_optional']
-            collection.hours_total_nonoptional = \
-                values['hours_total_nonoptional']
+                    values['tot_use_cases'] += 1
+                    values['hours_total_nonoptional'] = (
+                        values['hours_total'] - values['hours_total_optional'])
+                    # TODO: why doesn't this work?
+                    # collection.write(values)
+                    collection.hours_total = values['hours_total']
```

Comment:
as written above, `collection.write(values)` instead of the individual assignments